### ISSUE TITLE
App doesn't run on Catalina beta

### ISSUE THREAD

**Issue by mobitar** [standardnotes/desktop#399]
Actually, mixed reports on this one. Some users can, others can't.

**Comment by edas** [comment]
```
% open "/Applications/Standard Notes.app"
LSOpenURLsWithRole() failed with error -10810 for the file /Applications/Standard Notes.app
```

Standard Notes version 3.0.16
last Catalina beta 10.15 (19A578c)

**Comment by billymedia** [comment]
I just did a clean install of catalina, downloaded standardnotes and can confirm that standard notes wont open with the error reported by @edas .

**Comment by billymedia** [comment]
@mobitar any update on this? as it stands i have no access to any of my notes which is causing me issues. If you need me to provide any further info, please ask.

**Comment by billymedia** [comment]
Scratch that, installing 3.0.17 from the github releases page fixed it for me. Not sure how that differs from the version on standardnotes..org.

**Comment by chrisroos** [comment]
I have the same problem of it not working on Catalina (MacOS 10.15). I've tried the zip download from standardnotes.org and the dmg download from GitHub.

Opening the app extracted from the zip shows:

![Screenshot 2019-10-31 at 11 08 01](https://user-images.githubusercontent.com/6556/67942030-cea0f300-fbce-11e9-88c1-37b313ca3460.png)

Opening the app extracted from the dmg shows:

![Screenshot 2019-10-31 at 11 07 13](https://user-images.githubusercontent.com/6556/67942042-d8c2f180-fbce-11e9-9200-a9386cf05309.png)

**Comment by spipau** [comment]
I can confirm this as well, got a new MacBook Pro, updated to Catalina and downloaded v3.0.19.  Application is not starting and shows the same dialogue as @chrisroos post.

**Comment by eth3rgeist** [comment]
Same as @chrisroos post here, upgraded to Catalina, then tried to install the app. Not launching.

**Comment by yunesj** [comment]
Double clicking Standard Notes via the zip or the dmg gave me the same error others are reporting on 10.15. Right click opening Standard Notes allowed me to open it, but presented a warning the first time. Subsequent double clicking succeeded to open it.

**Comment by mllanes** [comment]
Same here with official catalina release:

`LSOpenURLsWithRole() failed with error -10810 for the file /Applications/Standard Notes.app`

**Comment by eth3rgeist** [comment]
Can confirm that „rightclick - open“ runs the app, still got a security warning but its dismisable, doesn‘t show up after second lunch. App starts normal after that.

**Comment by chrisroos** [comment]
Using right click and "Open" works for me too. Thanks @yunesj!

It displays the following warning on the first run but then opens normally after that if you choose "Open".

![Screenshot 2019-11-07 at 19 55 17](https://user-images.githubusercontent.com/6556/68422803-c7fb1880-0198-11ea-8729-0116e5558aac.png)

**Comment by mllanes** [comment]
Right click "open" still fails in Catalina 10.15.1

**Comment by mllanes** [comment]
Latest release in Github work as described above

**Comment by mobitar** [comment]
Fixed in 3.0.20.
